tests/composer/test_blueprints.py

```diff
@@ -233,7 +233,7 @@ def test_freeze(self):
     @unittest.skipUnless(os.path.exists("/run/weldr/api.socket"), "Test requires a running API server")
     def test_tag(self):
         """blueprints tag"""
-        rc = blueprints_tag("/run/weldr/api.socket", 0, ["glusterfs"], show_json=False)
+        rc = blueprints_tag("/run/weldr/api.socket", 0, ["example-http-server"], show_json=False)
```

Comment:
I see where this commit came from but I don't understand the reason behind it. Can't we use example-glusterfs.toml instead ?

Reply:
Well, the way I undertand it, the example-glusterfs doesn't exist anymore? I think it's a thing from the past.. These tests were skipped in travis all the time so I guess that's why this was not updated earlier..

Reply:
I do see `./tests/pylorax/blueprints/example-glusterfs.toml` which is what gets used by the test suite I believe so it definitely exists. Again, not sure if the name used in the code didn't exist how this test method even passed in Travis.

Reply:
I can try `example-glusterfs.toml` but I guess it would have to be uploaded first, I'm not sure now. Again this was skipped in travis so that's why this was not an issue until now.

Reply:
I changed it to `example-glusterfs` and it passed. So I guess it depends if there's any difference in which blueprints is used, test is passing as long as the blueprints exists it seems.

Reply:
ok, in this case the change is fine.